server/document.js

```diff
@@ -104,6 +104,17 @@ export class Head extends Component {
     const { page, pathname, buildId } = __NEXT_DATA__
     const pagePathname = getPagePathname(pathname)
 
+    let children = this.props.children
+    // show a warning if Head contains <title> (only in development)
+    if (process.env.NODE_ENV === 'development') {
```

Comment:
Let's use `if(process.env.NODE_ENV !== 'production')`, someone raised the issue that the `test` env should show these types of warnings.

Reply:
Will it be compiled away with `if(process.env.NODE_ENV !== 'production')` ?

(I'm trying to find the webpack plugin that does that ...)

Reply:
Yes, webpack automatically handles this since v4 when `mode: 'production'` it will add DefinePlugin set to `NODE_ENV=production`

Reply:
Nice 👌